packages/cozy-intent/package.json

```diff
@@ -50,7 +51,8 @@
     "lint": "eslint . --ext .ts,.tsx",
     "lint:fix": "yarn lint --fix",
     "prepublishOnly": "yarn build",
-    "test": "jest"
+    "test": "jest --config=./src/tests/jest.config.js",
+    "test:watch": "yarn test --watchAll"
```

Comment:
```suggestion
    "test:watch": "yarn test --watch"
```

Reply:
I recommend using https://jestjs.io/docs/cli#--watch

Reply:
Well it was required to have the coverage output shown during watch